frontend/apps/marketing/src/app/globals.css

```diff
@@ -8,4 +8,11 @@ body {
 body {
   margin: 0;
   padding: 0;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+footer {
+  margin-top: auto;
```

Comment:
Added this so the footer stays on the bottom of the page no matter how tall the content is.